en/protocol/image_transmission.md

```diff
@@ -0,0 +1,134 @@
+# Image Transmission Protocol
+
+> **Caution** This content has not been reviewed since being ported from the old website (and may be out of date). Updates/re-validation welcome!
+
+<span></span>
+> **Note** At time of writing (March 2018) the *QGroundControl* only uses the *Image Transmission Protocol* to show PX4 Flow images for focusing.
```

Comment:
Right, probably say that this should only be used for these kind of applications and not for general live video feeds. For that, RTSP streams are more applicable, and have far better performance and optimizations.